src/Bugsnag/Notifier.cs

```diff
@@ -39,39 +42,71 @@ public void Send(Event errorEvent)
                 Send(notification);
         }
 
+        public void SendStoredReports()
+        {
+            try
+            {
+                foreach (var storedReport in Store.ReadStoredJson())
+                {
+                    SendJson(storedReport);

```

Comment:
would it make sense to keep each `SentJson` in a try catch that way if sending one report fails it doesn't have a cascading effect on others?

Also, if this fails, will Bugsnag retry next app start or are the errors discarded?